public/splash.html

```diff
@@ -0,0 +1,138 @@
+<!-- Splash screen built upon vanilla html, js and css -->
+<html>
+  <head>
+    <script defer type="text/javascript">
+      // defer is for executing js before the page shows up
+      const queryString = window.location.search;
+      const urlParams = new URLSearchParams(queryString);
+
+      const primary = urlParams.get("primary");
+      const background = urlParams.get("background");
+      const display = urlParams.get("display");
+      const version = urlParams.get("version");
+
+      const root = document.documentElement;
+
+      // Notice: when passing url parameters, `#` is rstriped for unicode issues
+      root.style.setProperty("--primary", `#${primary}`);
+      root.style.setProperty("--background", `#${background}`);
+      root.style.setProperty("--display", `#${display}`);
+
+      window.onload = function () {
+        document.getElementById("version").innerHTML = `version ${version}`;
+      };
+    </script>
+    <style>
+      * {
+        user-select: none;
+      }
+      body {
+        background-color: var(--background);
+        overflow: hidden;
+      }
+      .root {
+        position: absolute;
+        width: 75%;
+        transform: translate(-50%, -66%);
+        left: 50%;
+        top: 50%;
+        display: flex;
+        flex-direction: column;
+      }
+      .title {
+        font-family: monospace;
```

Comment:
```suggestion
        font-family: "Rounded M+ 1p", sans-serif;
```
Using "Rounded M+ 1p" as title font.